### ISSUE TITLE
loop obj->method( ) ... is not triggered by v702

### ISSUE THREAD

**Issue by sbcgua** [abaplint/abaplint#768]
see discussion in https://github.com/larshp/abapGit/pull/3188

```abap
REPORT ZTEST105.

class lcl_app definition.
  public section.
    methods get_data
      returning
        value(rt_tab) type string_table.
endclass.

class lcl_app implementation.
  method get_data.
  endmethod.
endclass.


start-of-selection.

data lt_data type string_table.
field-symbols <i> like line of lt_data.

loop at lcl_app=>get_data( ) assigning <i>. " <<<<<<<<<<<<<<

endloop.
```

This is 7.4 syntax
